src/sentry/conf/server.py

```diff
@@ -817,12 +817,10 @@ def create_partitioned_queues(name):
     "organizations:custom-symbol-sources": True,
     # Enable the events stream interface.
     "organizations:events": False,
-    # Enable events v2 instead of the events stream
-    "organizations:events-v2": False,
```

Comment:
wondering if we need to remove these flags for `getsentry`

Reply:
The feature handler for events-v2 would only be called if a check for the events-v2 flag was done which won't happen anymore. I'll clean up the feature handlers too though.